.github/workflows/setup_release_testing/action.yaml

```diff
@@ -75,7 +75,7 @@ runs:
 
     - name: Install python packages & dependencies
       run: |
-        pip install runhouse[aws,gcp,kubernetes]
+        pip install git+https://github.com/run-house/runhouse.git@temp-launcher-testing#egg=runhouse[aws,gcp,kubernetes]
```

Comment:
Think it is better to run it on the most updated version of the client which is used by the launcher. Running it on the latest release won't reveal any new bugs, if such exist.